pom.xml

```diff
@@ -17,28 +17,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    
+
```

Comment:
Please use nacos code style to reformat
the indent has problem for now.

Reply:
done